assets/src/edit-story/app/index.js

```diff
@@ -52,9 +52,9 @@ function App({ config }) {
         <ConfigProvider config={config}>
           <APIProvider>
             <HistoryProvider size={50}>
-              <StoryProvider storyId={storyId}>
-                <FontProvider>
-                  <SnackbarProvider>
+              <SnackbarProvider>
```

Comment:
We don't need this here now do we? @ndev1991

Reply:
No, not anymore!

Reply:
Fixed